examples/Mediawiki/MakeUser.yaml

```diff
@@ -0,0 +1,95 @@
+# This demonstrates the "resource" feature, similar to fixtures.
+# See T228889
+
+resource: User
+description: Creates a user and logs them in
+type: RPC
+
+use-fixtures:
+  - RootUser
+
+variables:
+  name: p/uniq: "User_"
+  email: p/uniq: "bob@example"
```

Comment:
I'm not sure how strict Mediawiki is with emails. Can it be just about anything or must it end in a proper top-level domain?

Reply:
nor sure either. I'm pretty sure root@localhost is accepted (as it indeed should). Email address validation is a real adventure, <https://hackernoon.com/the-100-correct-way-to-validate-email-addresses-7c4818f24643>. But I can change my examples to example.com, for sanity :)